src/ray/ray_syncer/ray_syncer.cc

```diff
@@ -30,12 +30,14 @@ namespace ray::syncer {
 
 RaySyncer::RaySyncer(instrumented_io_context &io_context,
                      const std::string &local_node_id,
-                     RpcCompletionCallback on_rpc_completion)
+                     RpcCompletionCallback on_rpc_completion,
+                     bool batching_enabled)
```

Comment:
```suggestion
                     bool resource_update_batching_enabled)
```
nit: important because the `commands` channel is never batched

side note -- @Sparks0219 this is another indication that mixing the resource updates and commands channels feels really obtuse

Reply:
The COMMAND channel is actually batched as well, using the same logic to keep RaySyncer clean and generic. Would you prefer keeping this unified batching design, or should I revert to batching only RESOURCE_VIEW messages?

Reply:
Got it. This is fine as is IMO -- we are likely to remove `COMMANDS` from this path anyways. @Sparks0219 any objection?